hazelcast/src/main/java/com/hazelcast/client/impl/connection/tcp/TcpClientConnectionManager.java

```diff
@@ -130,6 +131,23 @@
 @SuppressWarnings({"checkstyle:MethodLength", "checkstyle:NPathComplexity"})
 public class TcpClientConnectionManager implements ClientConnectionManager, MembershipListener {
 
+    /**
+     * A private property to let users control the reconnection behavior of the client.
+     * <p>
+     * When enabled, the client will skip trying to connect the member list during
+     * reconnection attempts.
+     * <p>
+     * This property might be handy for users who are using the client with unisocket
+     * mode and exposing their multi-member cluster via a single load balancer or node port.
+     * In that scenario, the client would normally try to connect to the members from
+     * the member list first, but that would fail for sure, as the cluster members are not
+     * accessible from the client directly. In such use cases, setting this to
```

Comment:
```suggestion
     * In that scenario, the client would normally try to reconnect to the members in the last known
     * member list first after disconnection, but that would fail for sure, as the cluster members are not
     * accessible from the client directly. In such use cases, setting this to
```

Reply:
https://github.com/hazelcast/hazelcast/pull/24772/commits/6a9ddc35ce5e5c9abacccc9f340003fd05c9a64d